JetAnalyzers/bin/jet_synchfit_x.cc

```diff
@@ -25,194 +26,512 @@
 #include "JetMETAnalysis/JetUtilities/interface/JetInfo.hh"
 #include "JetMETAnalysis/JetAnalyzers/interface/REStyle.h"
 
-
 using namespace std;
 
 struct FitRes {
-      double etalowedge;
-      double etaupedge;
-      TF2 * fit;
+  double etalowedge;
+  double etaupedge;
+  TF2 * fit;
 };
 
-
 //===========================================================================
 // This method returns the graph from all the TProfile3D's
-TGraph2DErrors * getGraph2D(int iEta, const TProfile3D * prof, 
-                            const TProfile3D * profPt, 
-                            const TProfile3D * profRho){
-   
-   // The returning graph, with its name and title
-   TGraph2DErrors *graph  = new TGraph2DErrors();
-   graph->SetName(Form("p_offOverA_RhoVsJetPt_%d",iEta));
-   graph->SetTitle(Form("p_offOverA_RhoVsJetPt_%d;Rho;p_{T}^{pu};OffsetOverArea",iEta));
-   int nEvt = 0;
-   cout << "\tNBins (rho,pT):  (" << prof->GetYaxis()->GetNbins() << "," << prof->GetZaxis()->GetNbins() << ")" << endl;
-   //cout << "NBins rho:  " << prof->GetYaxis()->GetNbins() << endl;
-   //cout << "NBins pT:  " << prof->GetZaxis()->GetNbins() << endl; 
-   
-   for (int irho = 1 ; irho <= prof->GetYaxis()->GetNbins() ; irho ++){      
-      for (int irefpt = 1; irefpt <= prof->GetZaxis()->GetNbins() ; irefpt++){
-         
-         int Gbin = prof->GetBin(iEta, irho, irefpt);
-         nEvt += prof->GetBinEntries(Gbin);
-         // avoid points with empty content or too small error
-         if (prof   ->GetBinError  (iEta,irho,irefpt)  > 0.000001 &&
-             (fabs(prof->GetBinError(iEta,irho,irefpt)/prof->GetBinContent(iEta,irho,irefpt)))<0.5 &&// 0.3 ==> 0.5
-             (fabs(prof->GetBinError(iEta,irho,irefpt)/prof->GetBinContent(iEta,irho,irefpt)))>0.01 &&// 0.05 ==> 0.01
-             profPt ->GetBinContent(iEta,irho,irefpt)  > 0 &&
-             profPt ->GetBinError  (iEta,irho,irefpt)  > 0.1 &&
-             profRho->GetBinContent(iEta,irho,irefpt)  > 0 &&
-             profRho->GetBinError  (iEta,irho,irefpt)  > 0.1 ) {
-            
-            // get the relevant values
-            double rho  = profRho->GetBinContent(iEta, irho, irefpt); 
-            double rhoe = profRho->GetBinError  (iEta, irho, irefpt);
-            double pt   = profPt ->GetBinContent(iEta, irho, irefpt);
-            double pte  = profPt ->GetBinError  (iEta, irho, irefpt);
-            double ooa  = prof   ->GetBinContent(iEta, irho, irefpt);
-            double ooae = prof   ->GetBinError  (iEta, irho, irefpt);
-            
-            // Store the values
-            double n = graph->GetN(); 
-            graph->SetPoint(n, rho, pt, ooa);
-            graph->SetPointError(n, rhoe, pte, ooae);
-            
-         }//if
-         
+TGraph2DErrors * getGraph2D(int iEta, TProfile3D * prof,
```

Comment:
Please don't make white space changes unless absolutely necessary.

Reply:
Okay, I'll fix that.

Reply:
Please double-check. The diff of jet_synchfit_x.cc  is still very long and e.g. the getGraph2D still looks ~identical except for white space changes. Please make the formating such that one can better see the changes in the logic (if there are any). Thanks!

Reply:
Okay, I will double-check and try to see if I can make the format better!